examples/tensorrt_llm/multinode/srun_benchmark.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# This is one of the only variables that must be set currently. The image must contain
+# genai-perf installation.
+IMAGE_BENCH="${IMAGE_BENCH:-""}"
```

Comment:
This can default to a triton sdk container or something, right? `nvcr.io/nvidia/tritonserver:25.06-py3-sdk`?